### PR TITLE
wip: v2 updates

### PR DESCRIPTION
**Pull request by nisarhassan12**

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -89,25 +20,9 @@ const Hero = styled.div`
...
-                    />
-                    &nbsp;
-                </div>
-                &nbsp;
-            </div>
             <Hero>
-                <h1>Frictionless Open-Source Contributions</h1>
-                <p className="seconed">Setting up a local dev environment can be daunting. - All the projects below have automated their dev setup with <a href="https://www.gitpod.io" target="_blank">Gitpod.io</a>, a free online IDE for GitHub. <span style={{ fontWeight: 600 }}>Now your first Open-Source contribution is just one click away.</span></p>
+                <h1><strong>Frictionless</strong> Open-Source Contributions</h1>
+                <p>All projects below have automated their dev setup with <a href="https://www.gitpod.io" target="_blank">Gitpod</a>.</p>
```
Comment (ChristinFrohne) [inline src/components/Header.js:25]: This needs to be explained a bit more. What about:
"Setting up a local dev environment can be daunting. All projects below describe their dev environment as code with Gitpod. `<br>` Now you can start contributing with a single click."

File: src/components/FileIssue.js [hunk 2]
```diff
@@ -48,12 +51,15 @@ const Styled = styled.section`
 `
 const FileIssue = () => (
     <Styled>
-        <div>
-            <h2>Adding Your Project</h2>
-            <p><strong>You want to streamline the experience for your contributors?</strong> We're happy to help automate your project's dev environment.
-            Please file an issue with a link to your repository and we'll help you set it up. Once the experience is great we're happy to add your project to the list.</p>
+        <div className="card">
+            <h2>Adding My Project</h2>
```
Comment (ChristinFrohne) [inline src/components/FileIssue.js:55]: "Add My Project" sounds better

**Comment by nisarhassan12** [comment]
>  any reasons why you used Montserrat instead of Lato which is used in the mockup? Just curious, you can also keep it like this

Thanks, @ChristinFrohne Sorry that I ignored the font change in the mock-up and used `Montserrat` I think without a clear reason in mind(Not sure what I thought about it) but now I have changed it to `Lato` which looks nice.

File: src/components/Header.js (hunk 1 above)
Comment (nisarhassan12) [reply src/components/Header.js:25]: Thanks. This is how it looks with this change in place. I feel like the width of the paragraph is a bit long:

image

reducing the width with the `<br />` in place does not look good either:

image

Would it be ok if I were to get rid of the `<br />`? to me it looks better without the `<br />`:

image

**Comment by nisarhassan12** [comment]
@ChristinFrohne Please provide me with the URL for the 3rd tweet from Ancient Beast shown in the mock-up below. I was not able to find it myself.

image

File: src/components/Header.js (hunk 1 above)
Comment (ChristinFrohne) [reply src/components/Header.js]: Jep, fine by me  👍

**Comment by ChristinFrohne** [comment]
>@ChristinFrohne Please provide me with the URL for the 3rd tweet from Ancient Beast shown in the mock-up below. I was not able to find it myself.

You can either use the three existing quotes or add ancient beast as well. Whatever fits best. But we won't need a scrolling here as there are so few quotes 
https://twitter.com/AncientBeast/status/1194754176900902917

**Comment by nisarhassan12** [comment]
@ChristinFrohne I think using 3 existing ones is ok. I have updated the layout for the tweets please have a look again. Thanks

File: src/components/AboutGitpod.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                <div className="text">
+                    <h3>Introducing Gitpod — Frictionless Coding on GitLab, GitHub & Bitbucket</h3>
+                    <p>Every day developers waste millions of hours switching contexts between projects and branches, setting up development environments, or simply waiting for builds to complete.</p>
+                    <a
+                        href="https://www.gitpod.io/blog/gitpod-launch/"
+                        target="_blank"
+                        rel="noopener"
+                        className="btn"
+                    >
+                        Learn more about Gitpod
```
Comment (ChristinFrohne) [inline src/components/AboutGitpod.js:60]: We can shorten this to
"More about Gitpod"

**Comment by nisarhassan12** [comment]
>  Tweets: doesn't the ancient beast tweet fit perfectly in the current tweet layout. Please add it if you agree

Thanks, it does not fit correctly in the current layout due to the height of the contents this is how it looks:

image

**Comment by nisarhassan12** [comment]
@ChristinFrohne I have made the requested change please have a look whenever you'll have time. Thanks.
